Program.cs

```diff
@@ -11,30 +11,30 @@ public static void Main(string[] args)
 
             IList<Item> Items = new List<Item>{
                 new Item {Name = "+5 Dexterity Vest", SellIn = 10, Quality = 20},
-                new Item {Name = "Aged Brie", SellIn = 2, Quality = 0},
-                new Item {Name = "Elixir of the Mongoose", SellIn = 5, Quality = 7},
-                new Item {Name = "Sulfuras, Hand of Ragnaros", SellIn = 0, Quality = 80},
-                new Item {Name = "Sulfuras, Hand of Ragnaros", SellIn = -1, Quality = 80},
-                new Item
-                {
-                    Name = "Backstage passes to a TAFKAL80ETC concert",
-                    SellIn = 15,
-                    Quality = 20
-                },
-                new Item
-                {
-                    Name = "Backstage passes to a TAFKAL80ETC concert",
-                    SellIn = 10,
-                    Quality = 49
-                },
-                new Item
-                {
-                    Name = "Backstage passes to a TAFKAL80ETC concert",
-                    SellIn = 5,
-                    Quality = 49
-                },
-				// this conjured item does not work properly yet
-				new Item {Name = "Conjured Mana Cake", SellIn = 3, Quality = 6}
+                 new Item {Name = "Aged Brie", SellIn = 2, Quality = 0},
+                 new Item {Name = "Elixir of the Mongoose", SellIn = 5, Quality = 7},
+                 new Item {Name = "Sulfuras, Hand of Ragnaros", SellIn = 0, Quality = 80},
+                 new Item {Name = "Sulfuras, Hand of Ragnaros", SellIn = -1, Quality = 80},
+    //             new Item
+    //             {
+    //                 Name = "Backstage passes to a TAFKAL80ETC concert",
+    //                 SellIn = 15,
+    //                 Quality = 20
+    //             },
+    //             new Item
+    //             {
+    //                 Name = "Backstage passes to a TAFKAL80ETC concert",
+    //                 SellIn = 10,
+    //                 Quality = 49
+    //             },
+    //             new Item
+    //             {
+    //                 Name = "Backstage passes to a TAFKAL80ETC concert",
+    //                 SellIn = 5,
+    //                 Quality = 49
+    //             },
+				// // this conjured item does not work properly yet
```

Comment:
I see why you have commented this out, but for future you should avoid having blocks of commented code in your commits even if it will be fixed in next commits. You should just remove them. For now, it's fine

Reply:
have fixed in the next pull request